cli/run/index.ts

```diff
@@ -48,7 +48,7 @@ export default function runRollup(command: any) {
 		environment.forEach((arg: string) => {
 			arg.split(',').forEach((pair: string) => {
 				const [key, ...value] = pair.split(':');
-				if (value) {
+				if (value && value.length) {
```

Comment:
`if (value.length)` should be enough I guess as `value` must always be an array.